todo_app/app.py

```diff
@@ -2,19 +2,43 @@
 from todo_app.data.session_items import add_item, get_items
 
 from todo_app.flask_config import Config
+import requests
+import os
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    items = get_items()
+
+    url = "https://api.trello.com/1/boards/AbpSrXDQ/lists"
+
+    os.getenv("TRELLO_API_KEY")
+
+    querystring = {
+    "key":os.getenv("TRELLO_API_KEY"),
+    "token":os.getenv("TRELLO_API_TOKEN"),
+    "cards": "open" }
+
+    response = requests.request("GET", url, params=querystring)
+
+    response_json = response.json()
+
+    items = []
+
+    for trello_list in response_json:
+        for card in trello_list['cards']:
+            card['status'] = trello_list['name']
+            items.append(card)
```

Comment:
This works but could be improved. You've missed out the last part of the exercise - creating a "class" to represent the to-do items instead of continuing to just use dictionaries.

If you need some help getting started with classes you could look at https://corndelwithsoftwire.github.io/DevOps-Course-Python-Intro/07_classes.html 
But for this to-do app, we want a fairly simple `Item` class.

The difference won't be obvious but classes are an important concept to keep your code clean on a more complex project. The to-do item class will just be a way of grouping together data in a strictly defined object instead of a dictionary, which is less reliable - there can be arbitrary key/value pairs. If you were new to this project and wanted to know what the items in this list looked like, how would you? With a class there would be a clear, single source of truth.